examples/votchallenge/README.md

```diff
@@ -0,0 +1,93 @@
+# Running the votchallenge.net benchmarks against mosse-tracker
+
+These instructions are adapted from https://www.votchallenge.net/howto/tutorial_python.html
+
+All instructions assume that you are in this directory.
+
+## Installing vot tool in a python virtualenv
+
+First, you need to install the vot python tool, to run the benchmarks.
+
+At the time of writing, vot does not work with python 3.10 on macos (import error when starting up), so you may need to use an older version.
+
+```bash
+cd examples/votchallenge # if you are not already here
+
+python3.9 -m venv vot_venv
+source vot_venv/bin/activate
+pip install git+https://github.com/votchallenge/vot-toolkit-python
+```
+
+## Set things up for this project
+
+```bash
+cd examples/votchallenge # if you are not already here
+
+cargo build --release --example=votchallenge
+cp trackers.template.ini trackers.ini
+```
+
+Then change the last line of your new `trackers.ini`, to point at your
+`target/release/examples/votchallenge` executable. This must be an absolute path.
+
+## Check with a dummy sequence
+
+```bash
+cd examples/votchallenge && source vot_venv/bin/activate # if you are not already here
+
+vot test MosseRust
+```
+
+## Run the full benchmark suite 
+
+```bash
+cd examples/votchallenge && source vot_venv/bin/activate # if you are not already here
+
+vot test MosseRust
+```
+
+This only uses a couple of cores, and take around 30 minutes, so go make yourself a cup of tea. You should see output like this:
+
+```
+Downloading sequence dataset "VOT2020" with 60 sequences.
+ Downloading          |███████████████████████████████████████████████████████████████████████████| 100% [02:30<00:00]
+Download completed
+ Loading dataset      |███████████████████████████████████████████████████████████████████████████| 100% [00:00<00:00]
+Loaded workspace in '/Users/alsuren/src/mosse-tracker/examples/votchallenge'
+Found data for 1 trackers
+Evaluating tracker MosseRust
+ MosseRust/baseline   |███████████████████████████████████████████████████████████████████████████| 100% [13:24<00:00]
+ MosseRust/realtime   |███████████████████████████████████████████████████████████████████████████| 100% [13:11<00:00]
```

Comment:
there's no difference between our scores for these two. I *think* that they represent 1. and 2. in the paper, but I'm not sure. We're given them as bounding boxes because that's all we support. It might be better to use an older benchmark dataset that only deals in terms of bounding boxes, to give ourselves less of a handicap.

1. VOT-ST2020 challenge: This challenge was addressing short-term track-
ing in RGB images and has been running since VOT2013 with annual up-
dates and modifications. A significant novelty compared to 2019 was that
the target position was encoded by a segmentation mask.
2. VOT-RT2020 challenge: This challenge addressed the same class of track-
ers as VOT-ST2020, except that the trackers had to process the sequences in
real-time. The challenge was introduced in VOT2017. A significant novelty
compared to 2019 was that the target position was encoded by a segmenta-
tion mask.
3. VOT-LT2020 challenge: This challenge was addressing long-term track-
ing in RGB images. The challenge was introduced in VOT2018. The target
positions were encoded by bounding boxes.
4. VOT-RGBT2020 challenge: This challenge was addressing short-term
tracking in RGB+thermal imagery. This challenge was introduced in VOT2019
and can be viewed as evolution of the VOT-TIR challenge introduced in
VOT2015. The target positions were encoded by bounding boxes.
5. VOT-RGBD2020 challenge: This challenge was addressing long-term
tracking in RGB+depth (RGBD) imagery. This challenge was introduced
in VOT2019. The target positions were encoded by bounding boxes.

I've kind-of ignored the `unsupervis` results, because I'm not sure what they mean.